schema/me.js

```diff
@@ -0,0 +1,32 @@
+import date from './fields/date';
+import gravity from '../lib/loaders/gravity';
+import Profile from './profile';
+import {
+  GraphQLString,
+  GraphQLObjectType,
+} from 'graphql';
+
+const Me = new GraphQLObjectType({
+  name: 'Me',
+  fields: {
+    id: {
+      type: GraphQLString,
+    },
+    created_at: date,
+    email: {
+      type: GraphQLString,
+    },
+    profile: {
+      type: Profile.type,
+      resolve: ({ default_profile_id }) =>
+        gravity(`profile/${default_profile_id}`),
+    },
+  },
+});
+
+export default {
+  type: Me,
+  resolve: (root, options, { rootValue: { accessToken } }) => {
```

Comment:
So this is passed automatically from Force: https://github.com/artsy/force/blob/8c53c0889890af9c84fa646451c5cfb64a9211da/lib/metaphysics.coffee#L9-L10

You won't have to do anything on that end. When this server gets the request it just pulls it out and pushes it onto the rootValue: https://github.com/dzucconi/metaphysics/commit/1d892a8fb576db13c1edb458f71f07ba30f0ff46#diff-78c12f5adc1848d13b1c6f07055d996eR55